tests/testthat/test-unnest.R

```diff
@@ -162,3 +162,8 @@ test_that("grouping is preserved", {
   expect_equal(class(df), class(rs))
   expect_equal(dplyr::groups(df), dplyr::groups(rs))
 })
+
+test_that("column order is preserved", {
```

Comment:
Could you add another test for the case where you are unnesting a list of data frames?